src/post/PostSingleContent.scss

```diff
@@ -11,17 +11,21 @@
 
 .PostSingleContent__content,
 .PostSingleContent__replies .CommentBody {
-  font-size: 20px;
+  width: 820px;
```

Comment:
The point is that i dont want change the width, Medium use 700 width and doing fine, it doesnt matter is some article failed when it was made for Steemit width.